src/CMakeLists.txt

```diff
@@ -108,7 +108,7 @@ set(kafka_to_nexus_SRC
   MainOpt.cxx
   ${FMT_SRC}
   ${CMAKE_CURRENT_BINARY_DIR}/git_commit_current.cxx
-)
+  CLIOptions.cpp)
```

Comment:
Since we're using ``.cxx`` for all the source files, can we change the extension for this file as well?

Reply:
`.cpp` and `.h` are to be used according to LLVM guidelines.